x-pack/plugin/src/test/resources/rest-api-spec/api/enrich.execute_policy.json

```
@@ -6,7 +6,7 @@
       "paths": [
         {
           "path": "/_enrich/policy/{name}/_execute",
-          "methods": [ "PUT" ],
+          "methods": [ "PUT", "POST" ],
```

```suggestion
          "methods": [ "POST", "PUT" ],
```

SInce `POST` is preferred list it first.